ClientTest/src/com/rain/utils/android/robocop/clienttest/Main.java

```diff
@@ -11,6 +11,6 @@
 public class Main {
     public static void main(String[] args) {
         ContentProviderWriter writer = new ContentProviderWriter();
-        writer.createContentProvider("ClientTest/resources/example_schema.json", "ClientTest/src-gen/");
+        writer.createContentProvider("ClientTest/resources/classes_only_schema.json", "ClientTest/src-gen/");
```

Comment:
The name 'clienttest' is a bit of a misnomer.  This class is mainly used during development of the RoboCoP library to verify that the library works.  I don't think we need to create a new test class.